.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -9,7 +9,7 @@ addresses #
 
 
 <!-- Answer any that apply and delete the others. -->
-## What new work does this PR create?
+## What new issues will need to be opened because of this PR?
```

Comment:
`What new issues does this PR create?` conveys the same message. Isn't it better?

Reply:
hrm...

That depend on how you are using the word "issue".
I think for precision, we should only use "issue" to mean "numbered tasks tracked by github".

Reply:
Let's try this one and keep iterating if needed!